### PR TITLE
Improved graphical design of track selection

### PR DESCRIPTION
**Pull request by staubesv**
Hi @pmiddend 

I've worked on the graphical design for this PR. Please merge this PR soon.

### CODE REVIEW

**Comment by pmiddend** [comment]
Thanks @staubesv, I will take a look at it and merge it tomorrow!

**Comment by staubesv** [comment]
Thanks!

I've ended up with something like:

screen shot 2018-05-24 at 12 56 55

File: modules/admin-ui/src/main/webapp/styles/components/video/_video-editor.scss [hunk 1]
```diff
@@ -763,79 +763,67 @@ $segment-deleted-selected: darken($segment-deleted, 18%);
...
+    border: 1px solid #1169c5;
+    width: 690px;
+    height: 60px;
   }
 
   .video-canvas {
-    width: 320px;
-    height: 180px;
+    position: relative;
+    minwidth: 160px;
```
Comment (pmiddend) [inline modules/admin-ui/src/main/webapp/styles/components/video/_video-editor.scss]: I don't find `minwidth` mentioned anywhere. Typo?

Comment (staubesv) [reply modules/admin-ui/src/main/webapp/styles/components/video/_video-editor.scss]: Good catch... will update PR.

Comment (staubesv) [reply modules/admin-ui/src/main/webapp/styles/components/video/_video-editor.scss]: Fixed.

**Comment by pmiddend** [comment]
Okay, I've tested everything (upload, saving, GUI stuff) except the asset upload change. Merging...

**Pull request merged**